### DISCUSSION TITLE
MPD with Alexa

### DISCUSSION THREAD

**Discussion by stengland** [MusicPlayerDaemon/MPD#1471] [Show and tell]
Hello, not sure if anyone will find this useful but I have implemented a simple alaexa skill to control MPD. I use it to control MPD for my multiroom audio (using Snapcast). Enjoy!

https://github.com/stengland/alexa-mpd-skill

**Comment by MaxKellermann** [comment]
That's interesting (for people who find Alexa interesting, which excludes me) - do you want to add it to the MPD website? Add a PR for https://github.com/MusicPlayerDaemon/website/blob/master/content/clients/index.md

Note that you need a better license declaration. As it is now, your code is proprietary (not open source / free software).

**Reply by stengland** [reply]
🤦 Yep will do that later today.

**Comment by dvorpahl** [comment]
very good idea, Try it later
